pkg/tests/install/BUILD

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_test")
+load("//tests/util:defs.bzl", "fake_artifact")
+load("//:mappings.bzl", "pkg_attributes", "pkg_files", "pkg_mkdirs")
+load("//:install.bzl", "pkg_install")
+
+py_test(
+    name = "install",
```

Comment:
Doesn't buildifier complain if this is not called  "install_test" and the srcs are not "install_test.py"?

Reply:
Apparently not, but that's what the [style guide](https://docs.bazel.build/versions/main/skylark/build-style.html) recommends.

I haven't been doing this in any of my tests, but I'll do it starting now.